docs/man-openmpi/man1/mpirun.1.rst

```diff
@@ -532,11 +532,11 @@ generally useful to most Open MPI users:
   just before launch.
 
 * ``--launch-agent``: Name of the executable that is to be used to
-  start processes on the remote nodes. The default is ``PRRTEd``. This
+  start processes on the remote nodes. The default is ``prted``. This
   option can be used to test new daemon concepts, or to pass options
   back to the daemons without having mpirun itself see them. For
-  example, specifying a launch agent of ``PRRTEd -mca odls_base_verbose
-  5`` allows the developer to ask the ``PRRTEd`` for debugging output
+  example, specifying a launch agent of ``prted --mca odls_base_verbose
```

Comment:
Your example isn't quite correct here - it should be `--prtemca odls_base_verbose 5`.  Using the generic `mca` leaves open the possibility that `prted` will misinterpret the intent.